charts/client/chart.scss

```diff
@@ -870,22 +870,29 @@ $zindex-IndicatorDropdown: 50;
             position: relative;
             display: flex;
             align-items: center;
-            margin-right: 2 * $handle-width;
+            margin-left: 1%;
+            margin-right: 2 * $handle-diameter;
         }
 
         .handle {
-            height: 1em;
-            border-radius: 0.2em;
-            width: $handle-width;
+            height: $handle-diameter;
+            width: $handle-diameter;
+            border-radius: 100%;
             background: $controls-color;
             position: absolute;
             cursor: col-resize;
```

Comment:
Would `cursor: grab` or `cursor: grabbing` be better with the new handles?

Reply:
Ooh I'm in favor. @vanwoerden veto if needed

Reply:
Went ahead and did it for now.